compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/GraphicsLayerOwnerLayer.skiko.kt

```diff
@@ -176,6 +175,9 @@ internal class GraphicsLayerOwnerLayer(
         mutatedFields = scope.mutatedFields
         if (maybeChangedFields != 0 || outlineChanged) {
             invalidate()
+            if (layerManager.isArrEnabled) {
```

Comment:
The check existss inside. Please remove extra `if`s in this file

Reply:
I missed that, thanks. We can therefore remove the `isArrEnabled` from the interface.